shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-api/src/main/java/org/apache/shardingsphere/readwritesplitting/api/rule/ReadWriteSplittingDataSourceRuleTest.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.shardingsphere.readwritesplitting.api.rule;
```

Comment:
Sorry, also need to add license information at the top of the file, you can refer to other classes.